engine/api/src/main/scala/pl/touk/nussknacker/engine/api/definition/ParameterValidator.scala

```diff
@@ -57,7 +58,7 @@ case object NotBlankParameterValidator extends ParameterValidator {
     nodeId
   )
   private def isBlankStringLiteral(expression: String): Boolean =
-    BlankStringLiteralPattern.matcher(expression.trim).matches()
+    blankStringLiteralPattern.matcher(expression.trim).matches()
```

Comment:
Maybe instead of using Pattern we should use here RegExpParameterValidator? I think we should keep logic with pattern checking in only one class. It would be clearer.

Reply:
If we use RegExp then we will have another ErrorValidation type.. Then at FE will have problem with matching error: empty, regexp.. For Mandatory and NotBlank validators we should have specials error types.

Reply:
... also notblank have inverted logic - there is !matches instead of matches and it is surprisingly a big difference.